src/formatDate.js

```diff
@@ -50,7 +50,58 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const formatingArray = Array(3);
+
+  const separateOldFormat = fromFormat[fromFormat.length - 1];
```

Comment:
```suggestion
  const separateOldFormat = fromFormat[3];
```
тут можно явно указать